cg/meta/demultiplex/housekeeper_storage_functions.py

```diff
@@ -88,9 +89,13 @@ def add_sample_fastq_files_to_housekeeper(
             continue
 
         for sample_fastq_path in sample_fastq_paths:
+            fastq_file_path_appended: Path = append_flow_cell_name_to_fastq_file_path(
+                fastq_file_path=sample_fastq_path, flow_cell_name=flow_cell.id
+            )
+            rename_file(file_path=sample_fastq_path, renamed_file_path=fastq_file_path_appended)
             store_fastq_path_in_housekeeper(
                 sample_internal_id=sample_internal_id,
-                sample_fastq_path=sample_fastq_path,
+                sample_fastq_path=fastq_file_path_appended,
```

Comment:
This is a bug, if the file was previously renamed already we should not use the path returned from `append_flow_cell_name_to_fastq_file_path`

Reply:
fixed